app/javascript/src/components/Dashboard/Notes/Items/Item.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { Typography, Dropdown, Tag, Tooltip, Avatar } from "@bigbinary/neetoui";
+import { MenuVertical } from "neetoicons";
+
+import StatusText from "./StatusText";
+import { randomStatus } from "./utils";
+
+const Item = ({ note, editHandler }) => (
+  <div className="neeto-ui-shadow-s mb-4 border p-4">
+    <div className="flex justify-between border-b">
+      <div>
```

Comment:
Why this a `div` over a React Fragment?

Also, do we need this wrapper node? If yes, why?

Reply:
Getting a warning ```Passing a fragment to an HTML element is useless.``` when I tried fragment that's why used div

Reply:
Makes sense.